specifications/api/swagger.yaml

```diff
@@ -2755,6 +2858,47 @@ components:
             - <<: *transactionWithdrawalRequestSelf
             - <<: *transactionRedemptionRequest
 
+    ApiTransactionExtended: &ApiTransactionExtended
```

Comment:
Could we perhaps describe what we expect this transaction to look like? I can tell from this description that it's CBOR encoded, but what might the transaction look like before it's CBOR encoded? Do we expect it to be missing inputs and/or outputs?

Reply:
Just as someone with no context, I'm not sure how I might generate a transaction to call this endpoint.